cmd/fleetctl/preview.go

```diff
@@ -193,13 +196,42 @@ Use the stop and reset subcommands to manage the server and dependencies once st
 				return errors.Errorf("Failed to run docker-compose")
 			}
 
+			fmt.Println("Downloading standard query library")
+			buf, err := downloadStandardQueryLibrary()
+			if err != nil {
+				return errors.Wrap(err, "failed to download standard query library")
+			}
+
+			querySpecs, err := parseQuerySpecs(buf)
+			if err != nil {
+				return errors.Wrap(err, "failed to parse standard query library")
+			}
+
+			fmt.Println("Applying standard query library")
+			err = client.ApplyQueries(querySpecs)
+			if err != nil {
+				return errors.Wrap(err, "failed to apply standard query library")
+			}
+
 			fmt.Println("Preview environment complete. Enjoy using Fleet!")
 
 			return nil
 		},
 	}
 }
 
+func parseQuerySpecs(buf []byte) ([]*fleet.QuerySpec, error) {
+	queries, err := fleet.LoadQueryObjectsFromYaml(string(buf))
+	if err != nil {
+		return nil, err
+	}
+	querySpecs := make([]*fleet.QuerySpec, len(queries))
+	for i, q := range queries {
+		querySpecs[i] = &q.Spec
+	}
+	return querySpecs, nil
+}
+
```

Comment:
Let's please reuse https://github.com/fleetdm/fleet/blob/main/cmd/fleetctl/apply.go#L39 for the parsing.